packages/amplify-provider-awscloudformation/lib/transform-graphql-schema.js

```diff
@@ -139,6 +140,22 @@ async function migrateProject(context, options) {
   }
 }
 
+function loadCustomTransformersConfig(transformerConfigPath, context) {
```

Comment:
This is duplicated code, can't we reuse ```loadConfig``` from ```transformConfig.ts``` in ```graphql-transformer-core```? I miss the exist check for the file, BOM stripping, what we've at other places when reading JSON files.

Reply:
absolutely, missed that